source/includes/swift/_objects.md

```diff
@@ -13,12 +13,10 @@ curl -X GET \
 ```
 <code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/objects/:id/redirect</code>
 
-Preview private objects by generating a temporary URL that provides GET access for limited time. The generated URL will expire after one hour.
+Preview both public and private objects. Previews of private object will generate a temporary URL that provides GET access for limited time. The generated URL will expire after one hour.
```

Comment:
of private **objects**

Reply:
or "a" private object

Reply:
https://github.com/cloudops/cloudmc-api-docs/pull/246/commits/7d1e8e91c45f607f5cc84fc7595be021c81df8d7